src/server/config/complete.js

```diff
@@ -74,6 +74,8 @@ export default async function (kbnServer, server, config) {
     return kbnServer.config;
   });
 
+  server.decorate('server', 'core', kbnServer.core);
```

Comment:
__note:__ just a temporary location, need to find a better place to expose core.

Reply:
Yes, I used it as catch-all place for core -> legacy stuff (easy to grep for places that rely on core), do you think we should split into let's say `kbnServer.elasticsearch` and friends?

Reply:
No, I think this makes sense. When we pass plugin capabilities through, I'd assume we'd pass them as a sibling property to `core`, so it kind of mimics the `core`/`plugins` convention in the new platform.